### PR TITLE
Interactive footer & layout

### LINKED ISSUE #1192: Footer

### LINKED ISSUE #1196: Layout

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-07-31 at 15 36 01
screen shot 2017-07-31 at 15 36 12

Closes #1196 
Closes #1192

### CODE REVIEW

File: src/docs/components/DocsActiveBlock.jsx [hunk 1]
```diff
@@ -31,9 +31,11 @@ class DocsActiveBlock extends Component {
         }, {key: 'component'});
     }
 
+
     this.state = {
       showCode: null,
       changeBackground: this.props.backgroundColor || BACKGROUND_COLOR.LIGHT,
+      centeredItems: typeof this.props.centeredItems === 'undefined' ? true : this.props.centeredItems,
```
Comment (matikrk) [inline src/docs/components/DocsActiveBlock.jsx]: Do we need state, not props?
When you will be using this props You could use default value ({centeredItems=true})

File: src/docs/components/DocsActiveBlock.jsx [hunk 2]
```diff
@@ -85,6 +87,10 @@ class DocsActiveBlock extends Component {
       'docs-active-block--dark': this.state.changeBackground === BACKGROUND_COLOR.DARK
     });
 
+    const componentClass = classnames('docs-active-block__component', {
+      'docs-active-block__component--centered': this.state.centeredItems === true
```
Comment (matikrk) [inline src/docs/components/DocsActiveBlock.jsx]: not needed `=== true` if this will be bool

Comment (anna-wro) [inline src/docs/components/DocsActiveBlock.jsx]: changed

File: src/docs/components/DocsActiveBlock.jsx (hunk 1 above)
Comment (anna-wro) [inline src/docs/components/DocsActiveBlock.jsx]: I changed it, it's using only props now

**Pull request merged**